app/scripts/modules/core/application/config/serializeApplicationSection.directive.html

```diff
@@ -0,0 +1,18 @@
+<div class="row">
+  <div class="col-md-7">
+    <h4>Serialize Application</h4>
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <p>
+          Serializing the application will save the current state of the infrastructure using Terraform in a bucket. <br>
```

Comment:
Maybe word to "... of the infrastructure using [Terraform's config language](https://www.terraform.io/docs/configuration/index.html). in a bucket." (making sure to include that link).